reference/7.2/Microsoft.PowerShell.Core/About/about_ANSI_Terminals.md

```diff
@@ -71,11 +71,19 @@ The following members control how or when ANSI formatting is used:
 - `$PSStyle.OutputRendering` is a
   `System.Management.Automation.OutputRendering` enum with the values:
 
-  - **ANSI**: ANSI is always passed through as-is
+  - **ANSI**: This is the default behavior. ANSI is always passed through
+    as-is.
+
+    > [!IMPORTANT]
+    > You should use **ANSI** mode when redirecting output to a file or the
+    > pipeline that is intended to be executed downstream. This ensures that
+    > the output is not altered. Using any other mode alters the output, which
```

Comment:
I think you should explicitly say "Using any other mode alters the output by removing ANSI escape sequences, which..."